src/main/java/org/springframework/data/elasticsearch/client/InetSocketAddressParser.java

```diff
@@ -24,9 +24,10 @@
  * Utility to parse endpoints in {@code host:port} format into {@link java.net.InetSocketAddress}.
  *
  * @author Mark Paluch
+ * @author Andriy Redko
  * @since 3.2
  */
-class InetSocketAddressParser {
+public class InetSocketAddressParser {
```

Comment:
Making it public to be used by `org.springframework.data.elasticsearch.client.orhlc.DefaultClientConfiguration`